### PR TITLE
fix(modal): add large attribute

### PR DESCRIPTION
**Pull request by andrewdally**
## Description
Adds a `:large` attribute to modals. 

### Screenshots

|  before  |  after  |
|--------|--------|
|Screen Shot 2020-12-22 at 4 46 50 PM|Screen Shot 2020-12-22 at 4 45 49 PM|

### Steps for testing
1. Locally, you can go to http://localhost:6006/?path=/story/sage-modal--default
2. Check the `Large` knob and watch it grow 🌱 

## Related

-

### CODE REVIEW

**Comment by andrewdally** [comment]
I worked on this @QuintonJason, and he's going to finish adding the rails implementation to the sage-lib.

**Review by QuintonJason — APPROVED** [review]
💥

File: docs/app/views/examples/objects/modal/_markup1.html.erb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<dialog class="sage-modal sage-modal--large" data-js-modal="cool-modal-1" aria-modal="true">
```
Comment (AndrwM) [inline docs/app/views/examples/objects/modal/_markup1.html.erb:1]: Would you mind renaming `modal1` to `modal_large`?

**Review by AndrwM — APPROVED** [review]
This looks great +1

**Pull request merged**